2017/07/12 11:14:55 [error] 385#385: *117 FastCGI sent in stderr: "PHP message: PHP Warning:  Uncaught ErrorException: require(/srv/http/adzippy/routes/driver/protected_routes.php): failed to open stream: No such file or directory in /srv/http/adzippy/vendor/laravel/framework/src/Illuminate/Routing/Router.php:329
Stack trace:
#0 /srv/http/adzippy/vendor/laravel/framework/src/Illuminate/Routing/Router.php(329): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'require(/srv/ht...', '/srv/http/adzip...', 329, Array)
#1 /srv/http/adzippy/vendor/laravel/framework/src/Illuminate/Routing/Router.php(329): require()
#2 /srv/http/adzippy/vendor/laravel/framework/src/Illuminate/Routing/Router.php(285): Illuminate\Routing\Router->loadRoutes('/srv/http/adzip...')
#3 /srv/http/adzippy/vendor/laravel/framework/src/Illuminate/Routing/RouteRegistrar.php(104): Illuminate\Routing\Router->group(Array, '/srv/http/adzip...')
#4 /srv/http/adzippy/app/Providers/RouteServiceProvider.php(105): Illuminate\Routing\RouteRegistrar->group('/srv/http/adzip...')
#5 /srv/http/adzippy/app/...
PHP message: PHP Fatal error:  Illuminate\Routing\Router::loadRoutes(): Failed opening required '/srv/http/adzippy/routes/driver/protected_routes.php' (include_path='.:') in /srv/http/adzippy/vendor/laravel/framework/src/Illuminate/Routing/Router.php on line 329" while reading response header from upstream, client: 127.0.0.1, server: adzippy.dev, request: "GET /api/driver/user HTTP/1.1", upstream: "fastcgi://unix:/run/php-fpm/php-fpm.sock:", host: "adzippy.dev"
2017/07/12 11:15:21 [error] 385#385: *117 FastCGI sent in stderr: "PHP message: PHP Warning:  Uncaught ErrorException: require(/srv/http/adzippy/routes/driver/protected_routes.php): failed to open stream: No such file or directory in /srv/http/adzippy/vendor/laravel/framework/src/Illuminate/Routing/Router.php:329
Stack trace:
#0 /srv/http/adzippy/vendor/laravel/framework/src/Illuminate/Routing/Router.php(329): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'require(/srv/ht...', '/srv/http/adzip...', 329, Array)
#1 /srv/http/adzippy/vendor/laravel/framework/src/Illuminate/Routing/Router.php(329): require()
#2 /srv/http/adzippy/vendor/laravel/framework/src/Illuminate/Routing/Router.php(285): Illuminate\Routing\Router->loadRoutes('/srv/http/adzip...')
#3 /srv/http/adzippy/vendor/laravel/framework/src/Illuminate/Routing/RouteRegistrar.php(104): Illuminate\Routing\Router->group(Array, '/srv/http/adzip...')
#4 /srv/http/adzippy/app/Providers/RouteServiceProvider.php(105): Illuminate\Routing\RouteRegistrar->group('/srv/http/adzip...')
#5 /srv/http/adzippy/app/...
PHP message: PHP Fatal error:  Illuminate\Routing\Router::loadRoutes(): Failed opening required '/srv/http/adzippy/routes/driver/protected_routes.php' (include_path='.:') in /srv/http/adzippy/vendor/laravel/framework/src/Illuminate/Routing/Router.php on line 329" while reading response header from upstream, client: 127.0.0.1, server: adzippy.dev, request: "GET /api/driver/user HTTP/1.1", upstream: "fastcgi://unix:/run/php-fpm/php-fpm.sock:", host: "adzippy.dev"
2017/09/21 06:53:07 [error] 386#386: *76 open() "/srv/http/adzippy/public/images/call-01.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /images/call-01.png HTTP/1.1", host: "adzippy.dev", referrer: "http://adzippy.dev/"
2017/12/17 11:36:47 [error] 929#929: *52 open() "/srv/http/adzippy/public/images/pepe.jpg" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /images/pepe.jpg HTTP/1.1", host: "adzippy.dev"
2017/12/17 11:36:49 [error] 929#929: *52 open() "/srv/http/adzippy/public/images/pepe.jpg" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /images/pepe.jpg HTTP/1.1", host: "adzippy.dev"
2017/12/17 11:37:34 [error] 929#929: *56 open() "/srv/http/adzippy/public/images/pepe.jpg" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /images/pepe.jpg HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:20:01 [error] 455#455: *3 open() "/srv/http/adzippy/public/api/v1/driver/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:21:36 [error] 455#455: *4 open() "/srv/http/adzippy/public/api/v1/driver/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:21:49 [error] 455#455: *4 open() "/srv/http/adzippy/public/api/v1/driver/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:21:56 [error] 455#455: *5 open() "/srv/http/adzippy/public/api/v1/driver/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:35:35 [error] 455#455: *6 open() "/srv/http/adzippy/public/api/v1/driver/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:35:44 [error] 455#455: *7 open() "/srv/http/adzippy/public/api/v1/driver/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:35:45 [error] 455#455: *7 open() "/srv/http/adzippy/public/api/v1/driver/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:36:40 [error] 455#455: *7 open() "/srv/http/adzippy/public/api/v1/driver/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:36:42 [error] 455#455: *7 open() "/srv/http/adzippy/public/api/v1/driver/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:36:43 [error] 455#455: *7 open() "/srv/http/adzippy/public/api/v1/driver/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:36:44 [error] 455#455: *7 open() "/srv/http/adzippy/public/api/v1/driver/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:40:09 [error] 455#455: *12 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:40:13 [error] 455#455: *12 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:40:13 [error] 455#455: *12 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:40:13 [error] 455#455: *12 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:40:14 [error] 455#455: *12 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:40:14 [error] 455#455: *12 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:40:14 [error] 455#455: *12 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:40:14 [error] 455#455: *12 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:40:15 [error] 455#455: *12 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:40:25 [error] 455#455: *8 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:40:25 [error] 455#455: *8 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:42:35 [error] 455#455: *14 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:42:38 [error] 455#455: *14 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/02/06 12:44:59 [error] 455#455: *18 open() "/srv/http/adzippy/public/api/v1/driver/auth/get/profile/0104d03576.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /api/v1/driver/auth/get/profile/0104d03576.png HTTP/1.1", host: "adzippy.dev"
2018/03/21 05:02:33 [error] 423#423: *112 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 127.0.0.1, server: adzippy.dev, request: "POST /traffic/data/aksh89dsbsd09bs HTTP/1.1", upstream: "fastcgi://unix:/run/php-fpm/php-fpm.sock", host: "adzippy.dev"
2018/03/28 15:27:40 [error] 422#422: *57 open() "/srv/http/adzippy/public/advertiser/images/contact-01.png" failed (2: No such file or directory), client: 127.0.0.1, server: adzippy.dev, request: "GET /advertiser/images/contact-01.png HTTP/1.1", host: "adzippy.dev", referrer: "http://adzippy.dev/advertiser/register"
2018/04/12 14:47:01 [error] 423#423: *22 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 127.0.0.1, server: adzippy.dev, request: "GET /traffic/data?lat=28.415208&long=77.602501 HTTP/1.1", upstream: "fastcgi://unix:/run/php-fpm/php-fpm.sock", host: "adzippy.dev"
2018/04/12 14:47:09 [error] 423#423: *24 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 127.0.0.1, server: adzippy.dev, request: "GET /traffic/data?lat=28.785598&long=77.173347 HTTP/1.1", upstream: "fastcgi://unix:/run/php-fpm/php-fpm.sock", host: "adzippy.dev"
